_posts/blog/2018-10-17-ReviewGarcíaMárqu

```diff
@@ -0,0 +1,63 @@
+ 
```

Comment:
Please add a YAML header: http://www.halperta.com/criticalarchives/tutorial/writing-in-markdown/